src/App.tsx

```diff
@@ -10,13 +10,18 @@ import Register from "./pages/Register";
 import { ProtectedRoute } from "./components/Auth/ProtectedRoute";
 import PageLoader from "./pages/PageLoader";
 import Profile from "./pages/Profile";
-import { useUserContext } from "./context/UserContext";
 import "./App.css";
+import { RootState } from "./state/store";
+import { useDispatch, useSelector } from "react-redux";
+import { setUser } from "./state/slices/userSlice";
 import { PostContextProvider } from "./context/PostContext";
 
 function App() {
   const { isLoading, isAuthenticated, getAccessTokenSilently } = useAuth0();
-  const { user, setUser } = useUserContext();
+  // const { user, setUser } = useUserContext();
```

Comment:
Some comments we don't need, let's clean those up. There are some in `PostList.tsx` and `Register.tsx` too